indexer/osm_editor.hpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include "geometry/rect2d.hpp"
+
+#include "indexer/mwm_set.hpp"
+
+#include "std/function.hpp"
+#include "std/set.hpp"
+#include "std/string.hpp"
+
+struct FeatureID;
+class FeatureType;
+class Index;
+
+namespace osm
+{
+class Editor final
+{
+  struct Impl;
+  Impl * m_impl;
+
+  Editor();
+  ~Editor();
+  static Impl & GetImpl();
+
+public:
+  using TEmitterLambda = function<void(FeatureType &)>;
```

Comment:
Because std::function is not a lambda and generally can be bound to any callable object, ie functor.